cmd/agent/app/agent_test.go

```diff
@@ -93,4 +106,12 @@ func TestAgentStartStop(t *testing.T) {
 
 	agent.Stop()
 	assert.NoError(t, <-ch)
+
+	for i := 0; i < 1000; i++ {
+		if strings.Contains(logBuf.String(), "agent's http server exiting") {
```

Comment:
DRY the expected log?

Reply:
it is dry. If you're referring to L116, then it doesn't matter what that line says (I'll change it).